app/models/custom_field.rb

```diff
@@ -2,17 +2,18 @@
 #
 # Table name: custom_fields
 #
-#  created_at           :datetime         not null
-#  custom_field_set_id  :integer
-#  data_type            :string
-#  has_embeddable_media :boolean          default(FALSE), not null
-#  id                   :integer          not null, primary key
-#  internal_name        :string
-#  migration_position   :integer
-#  parent_id            :integer
-#  position             :integer
-#  required             :boolean          default(FALSE), not null
-#  updated_at           :datetime         not null
+#  created_at            :datetime         not null
+#  custom_field_set_id   :integer
+#  data_type             :string
+#  has_embeddable_media  :boolean          default(FALSE), not null
+#  id                    :integer          not null, primary key
+#  internal_name         :string
+#  migration_position    :integer
+#  override_associations :boolean          default(FALSE), not null
+#  parent_id             :integer
+#  position              :integer
+#  required              :boolean          default(FALSE), not null
+#  updated_at            :datetime         not null
```

Comment:
Blargh I hate when code tries to be cutsie and display things in tabular format like this and totally break diff review! All 12 of the above lines changed due to one actual change! 
</rant>